ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -784,5 +784,22 @@
     "EndButtonLabel": "Save and continue",
     "ShowEventNotes": "Y",
     "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "addOrRemoveLegalCounsel",
+    "Name": "Add or remove legal counsel",
+    "Description": "Add or remove legal counsel",
+    "DisplayOrder": 11,
+    "PreConditionState(s)": "Submitted;Gatekeeping;PREPARE_FOR_HEARING;FINAL_HEARING",
```

Comment:
Assuming it isn't required in the closed state but might be worth checking, can't see it being mentioned in the ticket anywhere

Reply:
I'm assuming that as well.
I've just asked the question. Let's see.

Reply:
It's been confirmed our assumptions were right.